cpp/src/text/detokenize.cu

```diff
@@ -52,8 +52,9 @@ struct detokenizer_fn {
   cudf::size_type const* d_row_map;          // indices sorted by output row
   cudf::size_type const* d_token_offsets;    // to each input token array
   cudf::string_view const d_separator;       // append after each token
-  cudf::size_type* d_offsets{};              // offsets to output buffer d_chars
+  cudf::size_type* d_sizes{};                // output sizes
   char* d_chars{};                           // output buffer for characters
+  cudf::detail::input_offsetalator d_offsets;
```

Comment:
Do we want a comment on this for consistency with the other members? (Applies in other files, too)